ssl/ssl_lib.c

```diff
@@ -4457,3 +4525,160 @@ int ssl_log_secret(SSL *ssl,
                           secret_len);
 }
 
+#define SSLV2_CIPHER_LEN    3
+
+int ssl_cache_cipherlist(SSL *s, PACKET *cipher_suites, int sslv2format,
+                         int *al)
+{
+    int n;
+
+    s->s3->send_connection_binding = 0;
```

Comment:
This arguably ought to be part of the SCSV processing, but I didn't want to change the order of processing more than needed, at first.